docs/multicluster/antctl.md

```diff
@@ -0,0 +1,28 @@
+# Antctl Multi-cluster command line
+
+Starting from version 1.6.0, Antrea supports the `antctl mc` commands, which can collect information from a leader cluster in a ClusterSet.
+This information is useful when trying to troubleshoot issues in an Antrea Multi-cluster ClusterSet.
```

Comment:
How about: ", for troubleshooting issues in an Antrea Multi-cluster ClusterSet".